VMEncryption/main/Utils/HandlerUtil.py

```diff
@@ -437,23 +437,22 @@ def archive_old_configs(self):
                     shutil.copy2(src, dest)
 
     def restore_old_configs(self):
-        if not os.path.exists(self.config_archive_folder):
-            return
-
-        for root, dirs, files in os.walk(self.config_archive_folder):
-            for file in files:
-                if file.endswith('.settings'):
-                    src = os.path.join(root, file)
-                    dest = os.path.join(self._context._config_dir, file)
-
-                    self.log("Copying {0} to {1}".format(src, dest))
-
-                    shutil.copy2(src, dest)
-
-                if file == 'mrseq':
-                    src = os.path.join(root, file)
-                    dest = os.path.join(os.path.join(self._context._config_dir, '..'), file)
-
-                    self.log("Copying {0} to {1}".format(src, dest))
-
-                    shutil.copy2(src, dest)
+        # restores all archived settings files carried over from prior extension versions 
+        # uses cp --preserve=timestamps for faster performance than shutil.copy2() 
```

Comment:
why did we add "--preserve=timestamps"?
Might be good to have that info here in the comments.